etc/skeleton/README.md.tpl

```diff
@@ -16,12 +16,13 @@ Development Environment
 Requirements:
 - `docker`
 - [`bblfsh-sdk`](https://github.com/bblfsh/sdk) _(go get -u github.com/bblfsh/sdk/...)_
+- UAST converter dependencies _(go get -t -v ./...)_
```

Comment:
That would be awesome, but for now [stretchr/testify](https://github.com/stretchr/testify) is missing so `mvn test` fails \wo it.